sympy/combinatorics/perm_groups.py

```diff
@@ -2027,6 +2027,8 @@ def is_solvable(self):
 
         """
         if self._is_solvable is None:
+            if self._order is not None and self._order % 2 != 0:
```

Comment:
Perhaps `self.order` could be used. That may take some time if it was not computed before, but computation of the derived series is also expensive.

Reply:
I have tried that but it was showing `TypeError` somehow `self.order` behaves as a method.
 ```
   if self.order is not None and self.order % 2 != 0:
TypeError: unsupported operand type(s) for %: 'method' and 'int'

```

Reply:
`self.order` is a method, so `self.order() % 2 != 0` should be used. (In particular, it is never `None`.)

Reply:
So sorry, I thought it's a property.